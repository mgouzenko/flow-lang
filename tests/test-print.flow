int main () {
  println("hey");
  println(2*3+4);
  println(3.3);
  println('c');
  print("hey");
  print(7);
  print('d');
  print(8.9);
}