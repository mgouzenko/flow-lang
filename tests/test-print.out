10
hello
3